### PR TITLE
Minor tweaks to help with getting setup

### PR DESCRIPTION
**Pull request by adambutler**
Fixed:

- Hard coded reference to `https://nexmo.ngrok.io` in `app.js`
- Added missing instruction to install `concurrently` npm module.

Changed:

- A few minor copy tweaks to the readme & `.env.example`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Before you begin you will need:
 Clone and install dependencies:
 
 ```bash
+npm install -g concurrently
```
Comment (leggetter) [inline README.md]: This should be in package.json as a Dev dependency rather than something you have to manually install.

File: index.js [hunk 2]
```diff
@@ -87,18 +87,18 @@ app.get('/answer', (req, res) => {
     {
       action: 'input',
       submitOnHash: true,
-      eventUrl: ['https://nexmo.ngrok.io/find-bin']
+      eventUrl: [`${BIN_BASE_URL}/find-bin`]
```
Comment (leggetter) [inline index.js]: 👍

File: .env-example [hunk 3]
```diff
@@ -1,3 +1,3 @@
-BIN_PHONE_NUMBER=
-APPLICATION_ID=6fb5c4d2-9a05-4041-8d86-14ac295b25f6
-BIN_BASE_URL=https://nexmo.ngrok.io
\ No newline at end of file
+BIN_PHONE_NUMBER=PHONE_NUMBER
```
Comment (leggetter) [inline .env-example]: Good consistency from the README 👍

**Review by leggetter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (adambutler) [inline README.md]: 👌

**Comment by adambutler** [comment]
@leggetter Changes made 👀

**Review by leggetter — APPROVED** [review]

**Pull request merged**